firmware_configurations/klipper/TechnoPhreak/BTT_SKR_v2/printer.cfg

```diff
@@ -0,0 +1,608 @@
+[extruder]
+step_pin: PD15
+dir_pin: !PD14
+enable_pin: !PC7
+##	Update value below when you perform extruder calibration
+##	If you ask for 100mm of filament, but in reality it is 98mm:
+##	rotation_distance = <previous_rotation_distance> * <actual_extrude_distance> / 100
+##  22.6789511 is a good starting point
+rotation_distance: 22.6789511	#Bondtech 5mm Drive Gears
+##	Update Gear Ratio depending on your Extruder Type
+##	Use 50:17 for Afterburner/Clockwork (BMG Gear Ratio)
+##	Use 80:20 for M4, M3.1
+gear_ratio: 50:17				#BMG Gear Ratio
+microsteps: 16
+full_steps_per_rotation: 200	#200 for 1.8 degree, 400 for 0.9 degree
+nozzle_diameter: 0.400
+filament_diameter: 1.75
+heater_pin: PB3
+##	Validate the following thermistor type to make sure it is correct
+sensor_type: ATC Semitec 104GT-2
+sensor_pin: PA2
+min_temp: 10
+max_temp: 270
+max_power: 1.0
+min_extrude_temp: 170
+control = pid
+pid_kp = 26.213
+pid_ki = 1.304
+pid_kd = 131.721
+##	Try to keep pressure_advance below 1.0
+#pressure_advance: 0.05
+##	Default is 0.040, leave stock
+#pressure_advance_smooth_time: 0.040
+
+##	E0 on MCU X/Y
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 extruder]
+uart_pin: PC6
+interpolate: false
+run_current: 0.5
+hold_current: 0.4
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+#####################################################################
+# 	Bed Heater
+#####################################################################
+
+[heater_bed]
+##	SSR Control - Z board, BED Pin
+heater_pin: z:PB7
+sensor_type: NTC 100K beta 3950
+sensor_pin: z:PA1
+##	Adjust Max Power so your heater doesn't warp your bed
+max_power: 0.6
+min_temp: 0
+max_temp: 120
+control: pid
+pid_kp: 58.437
+pid_ki: 2.347
+pid_kd: 363.769
+
+#####################################################################
+# 	Probe
+#####################################################################
+
+[probe]
+##	Inductive Probe
+##	This probe is not used for Z height, only Quad Gantry Leveling
+##	Z_MAX on mcu_z
+##	If your probe is NO instead of NC, change pin to !^z:PE4
+pin: ^z:PE4
+x_offset: 0
+y_offset: 25.0
+z_offset: 0
+speed: 10.0
+samples: 3
+samples_result: median
+sample_retract_dist: 3.0
+samples_tolerance: 0.006
+samples_tolerance_retries: 3
+
+#####################################################################
+# 	Fan Control
+#####################################################################
+
+[heater_fan hotend_fan]
+##	Hotend Fan - XYE board, FAN0 Connector
+pin: PB7
+max_power: 1.0
+kick_start_time: 0.5
+heater: extruder
+heater_temp: 50.0
+##	If you are experiencing back flow, you can reduce fan_speed
+#fan_speed: 1.0
+
+[fan]
+##	Print Cooling Fan - XYE board, FAN1 Pin
+pin: PB6
+kick_start_time: 0.5
+##	Depending on your fan, you may need to increase this value
+##	if your fan will not start. Can change cycle_time (increase)
+##	if your fan is not able to slow down effectively
+off_below: 0.10
+
+[heater_fan controller_fan]

```

Comment:
change this to `[controller_fan controller_fan]` and remove line 391. Controller fan is better because it turns the fans on when the steppers are engaged in order to better cool the stepper drivers.

Reply:
Should be good now